config/helm/schema.yaml

```diff
@@ -1,421 +0,0 @@
-# Copyright 2020 Dynatrace LLC
```

Comment:
Why did you delete `schema.yaml`?

Reply:
I could see the bitbucket pipeline being migrated in a way that doesn't require this workflow to be triggered on pull request or even be a separate workflow at all. Should I close this then?